iOS_SDK/OneSignalSDK/Source/OSMessagingController.m

```diff
@@ -475,11 +475,12 @@ - (void)evaluateMessageDisplayQueue {
 }
 
 /*
- Hide the window and call makeKeyWindow to ensure the IAM will not be shown
- */
+ Hide the top level IAM window
+ After the IAM window is hidden, iOS will automatically promote the main window
+ This also re-shows the keyboard automatically if it had focus in a text input
+*/
 - (void)hideWindow {
```

Comment:
this method I'm pretty sure is not needed, this was split on a new method to shadow it. You can have the line back to were is called